source/aggregation-pipeline-builder.txt

```diff
@@ -89,69 +89,211 @@ Limitations
+               :alt: Toggle on stage view mode 
+
+         .. step:: At the bottom of the aggregation pipeline pane, select the 
+            :guilabel:`+ Add Stage` button to open a new aggregation stage 
+            modal.
+
+         .. step:: On the upper-right corner of the stage modal, select the 
+            Focus Mode icon to open Focus Mode.
+
+            .. figure:: /images/compass/focus-mode-button.png 
+               :alt: Select the Focus Mode button
+         
+         .. step:: Click the :guilabel:`Select` dropdown and select the 
+            :manual:`aggregation pipeline stage 
+            </reference/operator/aggregation-pipeline/>` to use for the first 
+            stage of the pipeline. 
+
+         .. step:: Fill in your selected pipeline stage. 
+
+            You can adjust the width of the :guilabel:`Stage Input`, pipeline 
+            stage, and the :guilabel:`Stage Output` by dragging their border to 
+            the desired size.
+
+         .. step:: Click the :guilabel:`Add Stage` dropdown to add additional 
+            aggregation stages before or after your last aggregation stage. 
+            Repeat steps 1 and 2 for each additional stage.
+
+            You can add stages with the following keyboard shortcuts: 
+
+            - To add a stage after the current stage: 
+
+              - Windows / Linux: :guilabel:`Ctrl + Shift + A`
+
+              - Mac: :guilabel:`⌘ + Shift + A`
+            
+            - To add a stage before the current stage: 
+              
+              - Windows / Linux: :guilabel:`Ctrl + Shift + B`
+
+              - Mac: :guilabel:`⌘ + Shift + B` 
+
+         .. step::  **(Optional)** Toggle the previews for each stage to include 
+            or exclude them from the pipeline.
+               
+            .. include:: /includes/fact-agg-pipeline-preview-toggle.rst
+
+         .. step:: To navigate between different stages, select the stage you 
+            want to edit from the :guilabel:`Stage` dropdown in the upper-right 
+            corner of the Focus Mode modal. 
+
+            You can navigate between stages with the following keyboard 
+            shortcuts: 
+
+            - To go to the stage before the current stage: 
+
+              - Windows / Linux: :guilabel:`Ctrl + Shift + 9`
+
+              - Mac: :guilabel:`⌘ + Shift + 9`
+            
+            - To add a stage before the current stage: 
+              
+              - Windows / Linux: :guilabel:`Ctrl + Shift + 0`
+
+              - Mac: :guilabel:`⌘ + Shift + 0` 
+
+         .. step:: Click :guilabel:`x` to exit Focus Mode and select 
+            :guilabel:`Run` at the top right of the pipeline builder. 
+            |compass-short| returns your results in the document view. 
+            
+            .. warning:: 
+               
+               When you click :guilabel:`Run`, the aggregation runs against the 
+               entire collection.
```

Comment:
Could we explain a little bit about why this is a warning? What could be potentially be problematic about this behavior?

Reply:
After some investigation, it looks like this warning was originally added to differentiate the results that you get from the Stage Output preview (a limited sample of 10) and the final results from running the pipeline. This warning was likely to let users know that the stage previews they encountered previously in the stage editor isn't the full output of results. 

Opting to remove this warning but instead, adding a note in the "Fill in your pipeline stage" step.

Reply:
Ah yeah, that sounds right. Thanks for investigating and making the change. I think "warning" makes this look scarier than it actually is 😅